README.md

```diff
@@ -120,6 +121,62 @@ make run_tellus \
 This will build salus, tellus, and the guestvm then boot them with the
 system-installed qemu.
 
+## Building (using Bazel)
+
+```bash
+bazel build //:salus-all
+```
+
+## Running
+
+### Prerequisates
+
+Toolchains:
+
+- `bazelisk` see https://github.com/bazelbuild/bazelisk
+  - `bazelisk` will install the proper version of `bazel`
+  - `bazel` will install all the proper toolchains
+- `riscv64-unknown-elf-objcopy` on your path.
+
+QEMU, Linux kernel, Buildroot, and Debian:
+- The same as in Make/Cargo build
+
+### general
+
+
+What were make targets in the Make/Cargo build are now shell scripts.
+
+From the buildroot directory, run
```

Comment:
s/buildroot/top-level (or just say "Run:"). "buildroot" is an unrelated project.

Reply:
fixed